tests/switchmap_/server/db/table/test_macport.py

```diff
@@ -224,19 +227,18 @@ def test_update_row(self):
 
         # Do an update
         idx = result.idx_macport
+        original_row = _convert(result)
         updated_row = MacPort(
-            idx_l1interface=random.randint(1, db.TEST_MAXIMUM),
-            idx_mac=random.randint(1, db.TEST_MAXIMUM),
+            idx_l1interface=3,
+            idx_mac=7,
```

Comment:
We need a way to select random values that have not yet been selected. Find a way to do this. Hard coded values are not best practice.